src/pages/dashboard/index.page.tsx

```diff
@@ -688,43 +763,59 @@ const Dashboard = (): any => {
               title={t('dashboard.tabs.organizers')}
             >
               {tab === 'organizers' && (
-                <TabContent {...sharedTableContentProps} Row={OrganizerRow} />
+                <>
+                  <TabContent {...sharedTableContentProps} Row={OrganizerRow} />
+                  <Title>{t('dashboard.suggestions.title')}</Title>
+                  <Alert variant={AlertVariants.PRIMARY} marginY={4}>
+                    {t('dashboard.suggestions.description')}
+                  </Alert>
+                  <RequestOwnershipModal
+                    organizer={currentOrganizer}
+                    isVisible={isRequestModalVisible}
+                    onClose={() => setIsRequestModalVisible(false)}
+                  />
+                  <TabContent
+                    {...sharedTableContentProps}
+                    Row={SuggestedOrganizerRow}
+                    items={suggestedOrganizers.data?.member}
+                    status={suggestedOrganizers.status}
+                  />
+                </>
               )}
             </Tabs.Tab>
           </Tabs>
         </Stack>
         {i18n.language === 'nl' && <NewsletterSignupForm />}
         <Footer />
       </Page.Content>
-    </Page>,
-    <Modal
-      key="modal"
-      variant={ModalVariants.QUESTION}
-      visible={isModalVisible}
-      onConfirm={async () => {
-        UseDeleteItemByIdMutation.mutate({
-          id: parseOfferId(toBeDeletedItem['@id']),
-        });
-        setIsModalVisible(false);
-      }}
-      onClose={() => setIsModalVisible(false)}
-      title={t('dashboard.modal.title', {
-        type: t(`dashboard.modal.types.${tab}`),
-      })}
-      confirmTitle={t('dashboard.actions.delete')}
-      cancelTitle={t('dashboard.actions.cancel')}
-    >
-      {toBeDeletedItem && (
-        <Box padding={4}>
-          {t('dashboard.modal.question', {
-            name:
-              toBeDeletedItem.name[i18n.language] ??
-              toBeDeletedItem.name[toBeDeletedItem.mainLanguage],
-          })}
-        </Box>
-      )}
-    </Modal>,
-  ];
+      <Modal
+        variant={ModalVariants.QUESTION}
+        visible={isModalVisible}
+        onConfirm={async () => {
+          deleteItemByIdMutation.mutate({
+            id: parseOfferId(toBeDeletedItem['@id']),
+          });
+          setIsModalVisible(false);
+        }}
```

Comment:
This function doesn't need to be async I think

And setIsModalVisible(false); can probably be moved to a onSuccess or onSettled callback in the deleteItemByIdMutation hook?